tests/testthat/test-dupree_dir_integration.R

```diff
@@ -10,11 +10,16 @@ test_that("All .R files in subdirs are assessed by dupree_dir()", {
     file.path("R", "anRpackage-internal.R"),
     file.path("inst", "dir1", "R", "dont_dup_me.R")
   )
+  expect_is(
+    dupree_dir(package),
+    "dups",
+    info = "dupree_dir should return with class `dups`"
+  )
   expect_silent(
-    dupree_dir(package)
+    as.data.frame(dupree_dir(package))
```

Comment:
In `expect_silent` we just want `dupree_dir(args)` to run without error, the conversion to df is irrelevant here. Suggest: `expect_silent(dupree_dir(package))`